<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VideoOverviewWindow</class>
 <widget class="QMainWindow" name="VideoOverviewWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>524</width>
    <height>971</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Video Overview</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="OpenVideo">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>250</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Open Video</string>
    </property>
   </widget>
   <widget class="QLabel" name="MeanValueGraphDisplay">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>320</y>
      <width>301</width>
      <height>241</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>temp/blank.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="HistogramDisplay">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>580</y>
      <width>301</width>
      <height>241</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>temp/blank.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QSlider" name="FrameSlider">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>290</y>
      <width>311</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksAbove</enum>
    </property>
    <property name="tickInterval">
     <number>100</number>
    </property>
   </widget>
   <widget class="QLabel" name="VideoDisplay">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>351</width>
      <height>231</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>temp/blank.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>290</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>524</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>OpenVideo</sender>
   <signal>clicked()</signal>
   <receiver>VideoOverviewWindow</receiver>
   <slot>load_video()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>447</x>
     <y>282</y>
    </hint>
    <hint type="destinationlabel">
     <x>261</x>
     <y>485</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>FrameSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>VideoOverviewWindow</receiver>
   <slot>change_frame()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>215</x>
     <y>321</y>
    </hint>
    <hint type="destinationlabel">
     <x>261</x>
     <y>485</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
